x-pack/plugins/security_solution/README.md

```diff
@@ -0,0 +1,130 @@
+
+Make sure you're in the Kibana root directory.
+
+#### Endpoint API Integration Tests
+
+In one terminal, run:
+
+```bash
+INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:server --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+In another terminal, run:
+
+```bash
+INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:runner --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+#### Endpoint Functional Tests
+
+In one terminal, run:
+
+```bash
+INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:server --config x-pack/test/security_solution_endpoint/config.ts
+```
+
+In another terminal, run:
+
+```bash
+INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:runner --config x-pack/test/security_solution_endpoint/config.ts
+```
+
+### Running your own package registry
+
+If you are doing endpoint package development it will be useful to run your own package registry to serve the latest package you're building.
+To do this use the following commands:
+
+Make sure you're in the Kibana root directory.
+
+#### Endpoint API Integration Tests
+
+In one terminal, run:
+
+```bash
+PACKAGE_REGISTRY_OVERRIDE=<url to your package registry like http://localhost:8080> yarn test:ftr:server --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+In another terminal, run:
+
+```bash
+PACKAGE_REGISTRY_OVERRIDE=<url to your package registry like http://localhost:8080>  yarn test:ftr:runner --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+#### Endpoint Functional Tests
+
+In one terminal, run:
+
+```bash
+PACKAGE_REGISTRY_OVERRIDE=<url to your package registry like http://localhost:8080> yarn test:ftr:server --config x-pack/test/security_solution_endpoint/config.ts
+```
+
+In another terminal, run:
+
+```bash
+PACKAGE_REGISTRY_OVERRIDE=<url to your package registry like http://localhost:8080>  yarn test:ftr:runner --config x-pack/test/security_solution_endpoint/config.ts
+```
+
+### Using the default public registry
+
+If you don't have docker installed and don't want to run your own registry, you can run the tests using the ingest manager's default public package registry. The actual package registry used is [here](../../plugins/ingest_manager/common/constants/epm.ts#L9)
+
+Make sure you're in the Kibana root directory.
+
+#### Endpoint API Integration Tests
+
+In one terminal, run:
+
+```bash
+yarn test:ftr:server --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+In another terminal, run:
+
+```bash
+yarn test:ftr:runner --config x-pack/test/security_solution_endpoint_api_int/config.ts
+```
+
+#### Endpoint Functional Tests
+
+In one terminal, run:
+
+```bash
+yarn test:ftr:server --config x-pack/test/security_solution_endpoint/config.ts
+```
+
+In another terminal, run:
+
+```bash
+yarn test:ftr:runner --config x-pack/test/security_solution_endpoint/config.ts
```

Comment:
ahhhh - NOICE 👍 
so basically - no change as to what we (UI) do today. Just run it "normally". Thanks

Reply:
Yep exactly :)